src/EditorFeatures/Core.Wpf/InlineRename/CommandHandlers/RenameCommandHandler_RenameHandler.cs

```diff
@@ -49,13 +49,13 @@ public bool ExecuteCommand(RenameCommandArgs args, CommandExecutionContext conte
         {
             using (context.OperationContext.AddScope(allowCancellation: true, EditorFeaturesResources.Finding_token_to_rename))
             {
-                ExecuteRenameWorker(args, context.OperationContext.UserCancellationToken);
+                ExecuteRenameWorker(args, context);
             }
 
             return true;
         }
 
-        private void ExecuteRenameWorker(RenameCommandArgs args, CancellationToken cancellationToken)
+        private void ExecuteRenameWorker(RenameCommandArgs args, CommandExecutionContext context)
```

Comment:
>private void ExecuteRenameWorker(RenameCommandArgs args, CommandExecutionContext context) [](start = 8, length = 89)

The same question - not sure why this code is in a separate method.

Reply:
I am not sure why it is in separate method. but we kept code as they are since we don't know why. if someone can answer why it was done this way, or if people are okay merging it. then, we are happy to merge it.